elcid/urls.py

```diff
@@ -10,6 +10,7 @@
 from elcid import api
 
 from elcid import views
```

Comment:
What's this doing?

Reply:
It's just from the feb version  when we had the TB cliniclist view in urls.